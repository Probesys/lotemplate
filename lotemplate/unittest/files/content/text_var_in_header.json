{
  "my_var": {
    "type": "text",
    "value": "my replaced text"
  }
}