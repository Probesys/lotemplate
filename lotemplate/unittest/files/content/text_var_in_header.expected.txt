Can we change the following var my replaced text in the header ?
This document test the replacement of a variable in the header.
And in the footer the var “my replaced text” is still ther
